src/Stratis.Bitcoin/Connection/PeerLifetimeBehavior.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.Extensions.Logging;
+using NBitcoin.Protocol;
+using NBitcoin.Protocol.Behaviors;
+using Stratis.Bitcoin.Base;
+
+namespace Stratis.Bitcoin.Connection
+{
+    /// <summary>
+    /// A behaviour that will manage the lifetime of peers.
```

Comment:
just note that if we change naming and comments above (i think we should), we need to change this as well